woocommerce/woocommerce
# Add pre build script to delete assets

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/34663

This PR adds a prebuild script to first delete `css`, `js` files from `assets` so that there will not be any lingering files that can potentially be zipped up.






### How to test the changes in this Pull Request:



1. Checkout this branch
2. Run `pnpm run build --force`. 
3. Ensure all assets are generated in `plugins/woocommerce/assets`.
4. Now add a random file in `plugins/woocommerce/assets/css/test.test`.
5. Run `pnpm run build --force` again.
6. Check the file you created in step 4 is deleted. This is how you know the prebuild script ran and correctly cleared up the files.
7. CD `plugins/woocommerce` and run `pnpm run build:zip --force`.
8. Unzip the generated zipped file and ensure in `assets/css` and `assets/js` contain `css` and `js` files respectively.



### Other information:




### FOR PR REVIEWER ONLY:
